### ISSUE TITLE
Some yet unindetified tests produce output under the temp/ or output/ folders

### ISSUE THREAD

**Issue by ceki** [qos-ch/reload4j#15]
These test need to be identified and their output should be redirected to target/test-output

Configuration files are the place to start looking.

**Comment by noname713705** [comment]
Hello, I guess the following should help identify these tests.

`find src/test/ -type f -name "*.java" -print -exec grep ERFA {} \;`

```
src/test/java/org/apache/log4j/varia/ERFATestCase.java
public class ERFATestCase extends TestCase {
    public ERFATestCase(final String name) {
     * Send a message to the ERFA.
        erfa.setFile("output/ERFA-test2.log");
        assertTrue(new File("output/ERFA-test2.log").exists());
        assertTrue(new File("output/ERFA-test2.log.1").exists());
        assertTrue(new File("output/ERFA-test2.log.2").exists());
        assertFalse(new File("output/ERFA-test2.log.3").exists());
```

`find src/test/ -type f -name "*.java" -print -exec grep newdir {} \;`

```
src/test/java/org/apache/log4j/FileAppenderTest.java
            File newFile = new File("output/newdir/temp.log");
            File newDir = new File("output/newdir");
            wa.setFile("output/newdir/temp.log");
            assertTrue(new File("output/newdir/temp.log").exists());
```

` more temp` -> `org.apache.log4j.rewrite.RewriteAppenderTest`
```
INFO org.apache.log4j.rewrite.RewriteAppenderTest - p1: p2: Message 0
INFO org.apache.log4j.rewrite.RewriteAppenderTest - p1:Hello p2:World {p1=Hello, p2=World, x1=Mundo}
INFO org.apache.log4j.rewrite.RewriteAppenderTest - p1:Hello p2:World Message 1
```

And this:
`find src/test/ -type f -name "*.xml" -print -exec grep "\"temp\"" {} \;`

```
src/test/resources/org/apache/log4j/rewrite/map.xml
        <param name="file"   value="temp"/>
src/test/resources/org/apache/log4j/rewrite/property.xml
        <param name="file"   value="temp"/>
src/test/resources/org/apache/log4j/rewrite/reflection.xml
        <param name="file"   value="temp"/>
```

**Comment by ceki** [comment]
Fixed in 6645841828c8 and earlier commits
